projects/project-2/readings/Project 2 - SPARQL library

```diff
@@ -66,6 +78,10 @@ Detecting such unused properties helps maintain a clean and efficient ontology b
 
 4. Finding Classes with Conflicting Disjointness and Subclass Relationships
 
+Title: Disjoint Subclass Conflict
+Constraint Description: Identifies classes that are marked as disjoint but share a common superclass.
+Severity: Error
+
```

Comment:
Consider using rdfs:subClassOf+
to detect longer chains

Reply:
Thanks, I'll try :)